packages/vue/src/composables/useAuth.ts

```diff
@@ -15,20 +15,23 @@ export const useAuth = (serv?: InterpretService) => {
 };
 
 export const useAuthenticator = () => {
-  onMounted(() => {
-    watchEffect(() => {
-      if (!service.value) return;
-
-      const { state, send } = useAuth();
-
-      const facadeValues = getServiceFacade({ send, state: state.value });
-      for (const key of Object.keys(facade)) {
-        //@ts-ignore
-        useAuthenticatorValue[key] = facadeValues[key];
-      }
-      useAuthenticatorValue.send = send;
-      useAuthenticatorValue.state = state;
-    });
+  createValues();
+  watchEffect(() => {
+    createValues();
```

Comment:
I might refactor this again in the future, but this seems to work better, then surrounding in an an onMounted...